target/linux/realtek/patches-5.10/319-irqchip-irq-realtek-rtl-fix-VPE-affinity.patch

```diff
@@ -0,0 +1,145 @@
+--- a/drivers/irqchip/irq-realtek-rtl.c
++++ b/drivers/irqchip/irq-realtek-rtl.c
+@@ -28,6 +28,7 @@ static DEFINE_RAW_SPINLOCK(irq_lock);
+ 
+ #define REG(offset, cpu)	(realtek_ictl_base[cpu] + offset)
```

Comment:
you reccon we can avoid this somewhat nasty define? I suppose once we write a 'rtl-otto-irq' driver, that gets is address from the devicetree, this could somehow go away a that point ... It just looks horrendous, then again, most of our code here does ... meh

Reply:
Does anyone read "remove a nasty define" in the patch message?